CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # Changelog
+### 2.28.0
+This release includes:
+* Fluent Bit [1.9.7](https://fluentbit.io/announcements/v1.9.7/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.0
+* Amazon Kinesis Streams for Fluent Bit 1.10.0
+* Amazon Kinesis Firehose for Fluent Bit 1.7.0
+
+Compared to `2.27.0` this release adds:
+* Feature - Add gzip compression support for multipart uploads `S3 (Output)` plugin [multipart upload](https://docs.fluentbit.io/manual/pipeline/outputs/s3/)
```

Comment:
I prefer: Feature - Add gzip compression support for multipart uploads in [S3 Output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/s3/)

* no duplication of the phrase multipart uploads
* link is for S3 plugin docs not for S3 multipart API